WordPress/WordPressTest/Comments/Controllers/FullScreenCommentReplyViewControllerTests.swift

```diff
@@ -0,0 +1,142 @@
+
+import XCTest
+@testable import WordPress
+
+class FullScreenCommentReplyViewControllerTests: XCTestCase {
+    private var controller: FullScreenCommentReplyViewController?
+    private var window: UIWindow?
+
+    override func setUp() {
+        self.controller = FullScreenCommentReplyViewController.newEdit()
+        self.window = UIWindow()
+
+        guard let controller = self.controller else {
+            XCTFail("Test controller did not load correctly")
+            return
+        }
+
+        guard let window = self.window else {
+            XCTFail("Test window did not load correctly")
+            return
+        }
+
+        loadView(forController: controller, inWindow: window)
+    }
+
+    override func tearDown() {
+        window = nil
+        controller = nil
+
+        super.tearDown()
+    }
+
+    // MARK: - Tests
+
+    /// Tests loading a new instance of the view controller and checking the content
+    /// config property is being set correctly in the text view
+    func testTextViewContentIsLoaded() {
+        guard let controller = FullScreenCommentReplyViewController.newEdit() else {
+            XCTFail("Controller is nil")
+            return
+        }
+
+        let content = "Test Content"
+        controller.content = content
+
+        loadView(forController: controller, inWindow: UIWindow())
+        XCTAssertEqual(content, controller.textView.text)
+
+    }
+
+    /// Tests the button state becomes correctly disabled
+    /// as the `textView` content changes
+    func testReplyButtonStateEnables() {
+        guard let controller = self.controller else {
```

Comment:
Since we do this check in the `setUp` we can delete this `guard` block (and by making `controller` non-optional this test will be more readable!).